[WIP] Alternate observability pipeline image and architecture (#2687)
sensu/sensu-docs: content/sensu-go/6.0/observability-pipeline/_index.md

---
title: "Sensu Observability Pipeline"
linkTitle: "Observability Pipeline"
description: "Sensu's observability pipeline is a flexible, automated tool that gives you visibility into every part of your organization's infrastructure. Read this overview to learn how the observability pipeline works."
product: "Sensu Go"
version: "6.0"
weight: 10
layout: "single"
toc: false
menu:
  sensu-go-6.0:
    identifier: observability-pipeline
---

Sensu's observability pipeline is a flexible, automated tool that gives you visibility into every part of your organization's infrastructure.

> The Sensu [agent][1] is a lightweight client that runs on your [entities][2], which are the infrastructure components you want to observe.

Reviewer: ```suggestion
The Sensu [agent][1] is a lightweight process that runs on the infrastructure components you want to observe.
Each agent is represented in Sensu as an [entity][2].
```

If we feel it's important to address infra that we can't run an agent on, we might consider adding additional text like "Entities can also represent infrastructure which is observed by an agent acting as a proxy for entities which cannot run their own agent process." to explain proxy entities. I don't know if thats appropriate for this section.

> The Sensu [backend][1] schedules [checks][3] for agents to run on your entities based on the agent subscriptions you specify.

Reviewer: ```suggestion
The Sensu [backend][1] schedules [checks][3] for agents to run on your infrastructure.
Agents receive check execution requests based on the agent subscriptions you specify.
```

> The agent runs these checks on your entities to gather observation data about them.

Reviewer: ```suggestion
The agent runs these checks on your infrastructure to gather observation data about your networking, compute resources, applications, and more .
```
